server/src/main/java/com/hedera/block/server/producer/ProducerBlockItemObserver.java

```diff
@@ -76,11 +80,14 @@ public ProducerBlockItemObserver(
      * @param publishStreamRequest the PublishStreamRequest received from the upstream producer
      */
     @Override
-    public void onNext(@NonNull final PublishStreamRequest publishStreamRequest) {
-
-        @NonNull final BlockItem blockItem = publishStreamRequest.getBlockItem();
+    public void onNext(
+            @NonNull final com.hedera.hapi.block.protoc.PublishStreamRequest publishStreamRequest) {
 
         try {
+
+            @NonNull
```

Comment:
The method is annotated to return `@NonNull`, so there is no reason to _also_ annotate the local variable.